src/main/java/io/lbry/browser/DownloadManager.java

```diff
@@ -179,14 +170,15 @@ public void updateDownload(String id, String filename, double progress, double w
             builders.put(notificationId, builder);
         }
 
+        double progress = (writtenBytes / totalBytes) * 100;
         builder.setContentIntent(getLaunchPendingIntent(id, context))
                .setContentText(String.format("%.0f%% (%s / %s)", progress, formatBytes(writtenBytes), formatBytes(totalBytes)))
                .setGroup(GROUP_DOWNLOADS)
                .setProgress(MAX_PROGRESS, new Double(progress).intValue(), false)
                .setSmallIcon(android.R.drawable.stat_sys_download);
         notificationManager.notify(notificationId, builder.build());
 
-        if (progress == MAX_PROGRESS) {
+        if (progress >= MAX_PROGRESS) {
```

Comment:
I'm curious if this change was stylistic or necessary :grinning: (I agree the new one is better)

Reply:
I remember seeing a few cases where the value of `writtenBytes` was greater than `totalBytes`, so this ended up being necessary.